### ISSUE TITLE
Selection Info displays missing genes

### ISSUE THREAD

**Issue by cagatayTurkay** [Caleydo/caleydo#1505]
Also debugging shows that a selection delta with 1500 genes is fired.

![selectionissue](https://f.cloud.github.com/assets/4129319/818524/1bf6df26-ef96-11e2-96cd-186f4055e271.PNG)

**Comment by mstreit** [comment]
I assume that this is muli-mapping issue. If this is the case, there is no fix atm.

**Comment by sgratzl** [comment]
the following genes aren't mapable to DAVID:

```
C6ORF141, C15ORF48, C9ORF47, C20ORF103, ADH1B, ADH1A, C14ORF50, C16ORF74, APOBEC3B, DKFZP666G057, C8ORF47, HOTAIR, C8ORF48, C1ORF106, C8ORF4, C15ORF52, GLT8D4, C20ORF114, KIAA1881, FLJ21963, LRAP, C1ORF186, ACTBL1, C3ORF18, C20ORF174, C3ORF15, C1ORF135, C19ORF21, C14ORF161, C2ORF54, C4ORF18, C4ORF19, SMA4, OSTALPHA, C6ORF117, C1ORF59, C20ORF82, HIST1H4J, ACY1L2, C21ORF34, C6ORF211, C18ORF56, C2ORF40, LOC728378, UBE2E3, C1ORF115, CKMT1A, C1ORF116, C9ORF150, C1ORF64, C10ORF82, C10ORF81, C6ORF126, C15ORF42, HIST1H3H, C10ORF65, ZNF533, C21ORF81, POTE15, FAM77C, C1ORF34, C12ORF28, C4ORF31, C6ORF97, C5ORF23, C14ORF132, C4ORF39, C16ORF45, C5ORF29, CTA-246H3.1, C20ORF26, C9ORF128, ANKRD21, LOC100049076, GPR177, C8ORF13, C6ORF105, C10ORF140, IGLV2-14, C5ORF4, UNQ473, C11ORF70, C6ORF154, C20ORF39, ZNF659, C9ORF58, C9ORF68, FLJ22655, C9ORF66, IL6, C6ORF165, C1ORF168, C9ORF61, C20ORF42, HCG_1990170, C3ORF55, C3ORF52, C7ORF13, C5ORF38, C3ORF57, C10ORF30, C8ORF79, C13ORF33, C4ORF7
```

which is our primary id type for genes

-> can't fix

**Comment by mstreit** [comment]
will probably be fixed with #364
